src/libsync/configfile.cpp

```diff
@@ -951,7 +951,7 @@ void ConfigFile::setAutomaticLogDir(bool enabled)
 
 QString ConfigFile::logDir() const
 {
-    const auto defaultLogDir = QString(configPath() + QStringLiteral("/logs"));
+    const auto defaultLogDir = QString(QStandardPaths::writableLocation(QStandardPaths::AppLocalDataLocation) + QStringLiteral("/logs"));
```

Comment:
@franga2000 I'd say, you need to change the corresponding part in `QString ConfigFile::configPath() const` if you want to only affect Linux. There is certain logic in `QString ConfigFile::configPath() const` that gets executed for Windows. Better try not to change the behavior.

Reply:
@franga2000 Also, make sure **debug logs archive** can be generated via **Settings** dialog UI after this change.